scripts/update_guestbook.py

```
@@ -33,8 +33,11 @@ def parse_message(body: str) -> str:
         # フォールバック: 本文全体の最初の非空行
         lines = [l.strip() for l in body.splitlines() if l.strip() and not l.startswith("#")]
         msg = lines[0] if lines else "(no message)"
-    # テーブル内で問題になる文字をエスケープ
-    msg = msg.replace("|", "\\|").replace("\n", " ")
+    # テーブル内で問題になる文字をエスケープ。
+    # str.splitlines() は \r や   でも行を割るので、あとで既存行を読み直す
```

Ruff (0.16.1)

[warning] Comment contains ambiguous ` ` (LINE SEPARATOR). Did you mean ` ` (SPACE)?

(RUF003)